SEC/Securities/SecuritiesIssuance.rdf

```diff
@@ -547,6 +615,14 @@
 		<skos:definition>indicates a period of time in which investors can commit to purchase shares (or units) to be issued</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-sec-sec-iss;isIssuedInForm">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;isEvidencedBy"/>
+		<rdfs:label>is issued in form</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Security"/>
+		<rdfs:range rdf:resource="&fibo-sec-sec-iss;SecurityForm"/>
+		<skos:definition>indicates the form in which the security is issued, typically in registered form</skos:definition>
+	</owl:ObjectProperty>
+	
 	<owl:DatatypeProperty rdf:about="&fibo-sec-sec-iss;isOverAlloted">
```

Comment:
typo should be isOverAllotted (double t)

Reply:
ok